config/locales/en.yml

```diff
@@ -649,6 +649,7 @@ en:
         title: MP3 Not Provided
       processing:
         hint: Processing
+        still_processing: Processing is taking longer than expected. Thank you for waiting.
```

Comment:
Nice, I do like how this isn't highlighted in red now.   Looks less angry:

![image](https://github.com/user-attachments/assets/4504f19b-8fc4-4b1d-a9ba-9de4f73f5fcf)

I might prompt them that it's cool to click the retry button though... "Processing is taking longer than expected - you can continue waiting or retry"

Reply:
FWIW: I prefer the term "patience" over "waiting", ie: "thank you for your patience"... Throw in a llittle "Ope" and we might even get the user to chuckle in another wise frustrating moment.

I suggest:
"Ope, file processing is taking longer than expected - you can continue waiting or retry"